### DISCUSSION TITLE
Is it possible to translate/normalize the meshes in urdf file?

### DISCUSSION THREAD

**Discussion by zubair-irshad** [haosulab/SAPIEN#147] [Q&A]
Is it possible to have control over mesh translation recentering in SAPIEN? For reference, please see the screenshot below where object bounds are as follows and I need it to be centered in a unit box for rendering purposes. 
```
max_bounds [0.2323547  0.31434572 0.34190416]
min_bounds [-0.23751283 -0.22038734 -0.01534534]
```

Also is there a way to get these bounds directly from a loaded mesh? I got these from the object pointcloud created from multiple views. Thanks in advance for your help!

![Screenshot from 2023-12-10 20-58-31](https://github.com/haosulab/SAPIEN/assets/32943733/872f0f89-35cd-42e7-a916-f41d1f1856e9)

**Comment by fbxiang** [comment]
A bounding box API will be directly exposed in the next major release (SAPIEN 3). For SAPIEN 2, the best way to obtain bounding boxes is by querying the visual shapes as seen in ManiSkill 2
https://github.com/haosulab/ManiSkill2/blob/d04c2dc5bfd826172b2f3193cbf232a4a33bdd9e/mani_skill2/utils/geometry.py#L87-L99
